mds/api/auth.py

```diff
@@ -1,8 +1,11 @@
 """
 Authentication module for MDS API calls.
 """
-
+from mds.api.auth import AuthorizationToken
+import mds
+import mds.providers
```

Comment:
These imports, as with `json` below, are unused.

Reply:
It is corrected.